extensions/wikia/DesignSystem/services/DesignSystemGlobalNavigationService.class.php

```diff
@@ -34,7 +34,28 @@ public function linkDropdown() {
 	}
 
 	public function linkAuthentication() {
-		$this->setVal( 'model', $this->getVal( 'model' ) );
+		$model = $this->getVal( 'model' );
+		$messageKey = $model['title']['key'];
+		$href = $model['href'];
+		$classNames = '';
+
+		switch ( $messageKey ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS131)